frontend/src/scenes/scenes.ts

```diff
@@ -530,136 +530,138 @@ export const redirects: Record<
     '/settings/organization-rbac': urls.settings('organization-roles'),
 }
 
-export const routes: Record<string, Scene> = {
-    [urls.dashboards()]: Scene.Dashboards,
-    [urls.dashboard(':id')]: Scene.Dashboard,
-    [urls.dashboardTextTile(':id', ':textTileId')]: Scene.Dashboard,
-    [urls.dashboardSharing(':id')]: Scene.Dashboard,
-    [urls.dashboardSubcriptions(':id')]: Scene.Dashboard,
-    [urls.dashboardSubcription(':id', ':subscriptionId')]: Scene.Dashboard,
-    [urls.createAction()]: Scene.Action,
-    [urls.duplicateAction(null)]: Scene.Action,
-    [urls.action(':id')]: Scene.Action,
-    [urls.ingestionWarnings()]: Scene.DataManagement,
-    [urls.insightNew()]: Scene.Insight,
-    [urls.insightEdit(':shortId' as InsightShortId)]: Scene.Insight,
-    [urls.insightView(':shortId' as InsightShortId)]: Scene.Insight,
-    [urls.insightSubcriptions(':shortId' as InsightShortId)]: Scene.Insight,
-    [urls.insightSubcription(':shortId' as InsightShortId, ':itemId')]: Scene.Insight,
-    [urls.alert(':shortId')]: Scene.SavedInsights,
-    [urls.alerts()]: Scene.SavedInsights,
-    [urls.insightAlerts(':shortId' as InsightShortId)]: Scene.Insight,
-    [urls.insightSharing(':shortId' as InsightShortId)]: Scene.Insight,
-    [urls.savedInsights()]: Scene.SavedInsights,
-    [urls.webAnalytics()]: Scene.WebAnalytics,
-    [urls.webAnalyticsCoreWebVitals()]: Scene.WebAnalytics,
-    [urls.actions()]: Scene.DataManagement,
-    [urls.eventDefinitions()]: Scene.DataManagement,
-    [urls.eventDefinition(':id')]: Scene.EventDefinition,
-    [urls.eventDefinitionEdit(':id')]: Scene.EventDefinitionEdit,
-    [urls.propertyDefinitions()]: Scene.DataManagement,
-    [urls.propertyDefinition(':id')]: Scene.PropertyDefinition,
-    [urls.propertyDefinitionEdit(':id')]: Scene.PropertyDefinitionEdit,
-    [urls.dataManagementHistory()]: Scene.DataManagement,
-    [urls.database()]: Scene.DataManagement,
-    [urls.activity(':tab')]: Scene.Activity,
-    [urls.replay()]: Scene.Replay,
+export const routes: Record<string, [Scene, string]> = {
```

Comment:
This `sceneKey` is pretty convenient, but now if I were to add a new route to `routes`, I don't know what to really put in the second tuple element – it's kind of cryptic, there's no description for the team to know

Reply:
Let's find a way to make it more obvious. Currently I just reused the name of the url, however I think this could just be a freeform string that will be free for you to set within the scene. Especially since the `urls.ts` file will be build per-product now.